src/backend/distributed/sql/udfs/any_value/9.1-1.sql

```diff
@@ -1,7 +1,13 @@
+DO $proc$
```

Comment:
> Say you are on Citus 12.0/PG15, you already have any_value defined. Now, upgrade to Citus 12.1 then PG16. Wouldn't that fail?

ok, we handle than in pre/post_pg_upgrade scripts?

The first question is still valid

Reply:
> We normally do not change the old version files, they are already shipped to the clusters and most likely won't have any impact. aren't we supposed to create a new file, any_value/citus/12.1-1.sql or such?

For new clusters, they will get directly the correct one. Current any_value can't be created in any way, so I would either have to update `any_value/9.1-1.sql/`, or create `any_value/12.1-1.sql/` and update `citus--9.0-2--9.1-1.sql` to not include `"udfs/any_value/9.1-1.sql"` but to include `"udfs/any_value/12.1-1.sql"`. I would rather not touch the upgrade script so I chose to directly update `any_value/9.1-1.sql/`.

For existing clusters, once they update to 12.1-1, they will go through pre/post pg upgrade scripts as well because I included them in `citus--12.0-1--12.1-1.sql`.

I mostly referred to this as an example where something similar was done for PG14 support https://github.com/citusdata/citus/commit/8ef94dc1f503d14451f65d1574973bfc3840b7c1

Reply:
So, if you are an old user, and upgrade to Citus 12.1, we protect you via pre/post pg_upgrades.

Also, you cannot use PG 16 with Citus 9.1. Why do we need to change 9.1 sql file? Can't we just leave as-is and have 12.1 file which doesn't create the function for PG16+?

Reply:
> Also, you cannot use PG 16 with Citus 9.1. Why do we need to change 9.1 sql file?

I guess `make install all` will go through `citus--9.0-2--9.1-1.sql` no matter what.